### PR TITLE
fix: make eslint work with prettier

### PR DESCRIPTION
**Pull request by onuriltan**
Referenced by [the article](https://www.robinwieruch.de/prettier-eslint) with title: HOW TO COMBINE ESLINT AND PRETTIER

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -25,7 +25,7 @@ const bootServer = async () => {
     res.send('Hello World!');
   });
 
-  app.use('/api/task', task);
+  app.use('/aspi/task', task);
```
Comment (akocakaya) [inline src/index.ts]: aspi?

Comment (onuriltan) [reply src/index.ts]: 🤦

Comment (onuriltan) [reply src/index.ts]: @akocakaya fixed

**Review by akocakaya — APPROVED** [review]
you have my approve

**Pull request merged**